oap-server/analyzer/agent-analyzer/src/main/java/org/apache/skywalking/oap/server/analyzer/provider/trace/parser/listener/MultiScopesAnalysisListener.java

```diff
@@ -133,7 +133,7 @@ public void parseEntry(SpanObject span, SegmentObject segmentObject) {
             sourceBuilder.setSourceNormal(false);
             sourceBuilder.setDestServiceInstanceName(segmentObject.getServiceInstance());
             sourceBuilder.setDestServiceName(segmentObject.getService());
-            sourceBuilder.setDestLayer(Layer.GENERAL);
+            sourceBuilder.setDestLayer(fromSpanLayerValue(span.getSpanLayer()));
```

Comment:
I think this is wrong for MQ case. Entry span has the MQ as layer but it represents the service itself.

In my idea, you should not use `fromSpanLayerValue` directly, we just should consider `FAAS` service as a new general condition here, but all others should keep in `Layer.GNERAL`.

Reply:
done